public/keycloak/FetchConfig.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Fetches config from environment and API.
+ * @returns A promise to get & set session storage keys with appropriate values.
+ */
+export default async function fetchConfig (): Promise<any> {
+    // get config from environment
+    const origin: string = window.location.origin
```

Comment:
Why is this explicitly typed as a string but not the others below?

Reply:
Others also explicitly typed as a string.

Reply:
Thanks for making it consistent for all 3 strings.